doc/classes/CanvasItem.xml

```diff
@@ -456,6 +456,12 @@
 				Returns this item's transform in relation to the viewport.
 			</description>
 		</method>
+		<method name="get_visibility_layer_bit" qualifiers="const">
+			<return type="bool" />
+			<param index="0" name="layer" type="int" />
+			<description>
+			</description>
+		</method>
```

Comment:
Docs will need to be filled in

Reply:
Ah, they were deleted in the property name change ><